pkg/operator/ceph/file/dependent.go

```diff
@@ -23,17 +23,32 @@ import (
 	v1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
 	"github.com/rook/rook/pkg/clusterd"
 	"github.com/rook/rook/pkg/daemon/ceph/client"
+	"github.com/rook/rook/pkg/util"
 	"github.com/rook/rook/pkg/util/dependents"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const subvolumeGroupDependentType = "filesystem subvolume groups that contain subvolumes (could be from CephFilesystem PVCs or CephNFS exports)"
+
 // CephFilesystemDependents returns the subvolume group(s) which exist in the ceph filesystem that should block
 // deletion.
-func CephFilesystemDependents(clusterdCtx *clusterd.Context, clusterInfo *client.ClusterInfo, filesystem *v1.CephFilesystem) (*dependents.DependentList, error) {
+var CephFilesystemDependents = cephFilesystemDependents
+
+// with above, allow this to be overridden for unit testing
+func cephFilesystemDependents(clusterdCtx *clusterd.Context, clusterInfo *client.ClusterInfo, filesystem *v1.CephFilesystem) (*dependents.DependentList, error) {
```

Comment:
Moving `CephFilesystemDependents` to a var right above the function allows "peeking" in VSCode to see the function more easily (right below the var) than having it in a different file.